### PR TITLE
fix(Core/Item): Fix TrinketUpdate

### PR DESCRIPTION
**Pull request by pklloveyou**
## Changes Proposed:
- It has been tested on my server

## Issues Addressed:
- [Closes ](https://github.com/azerothcore/azerothcore-wotlk/pull/11487)

## SOURCE:
- [Closes ](https://github.com/azerothcore/azerothcore-wotlk/pull/11487)

## Tests Performed:
- Create mage
- Use the items in the picture
```
.ad 54588
.ad 54572
.ad 50648
.ad 50348
.ad 50353
.ad 50400
.ad 50399
.ad 49992
```
image

## How to Test the Changes:
- After using this PR, the spell will now be triggered

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1651231168158458211.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1651231168158458211');
```
Comment (IntelligentQuantum) [inline data/sql/updates/pending_db_world/rev_1651231168158458211.sql:1]: delete this line

**Review by IntelligentQuantum — CHANGES_REQUESTED** [review]

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Maelthyrr** [comment]
What is the difference with https://github.com/azerothcore/azerothcore-wotlk/pull/11487 ?

**Comment by pklloveyou** [comment]
I need to pull the code again, so close it

**Comment by Maelthyrr** [comment]
Tested and work.

**Comment by Nyeriah** [comment]
@pklloveyou why close?

**Pull request merged**